### PR TITLE
Add Voltage Control Reference Flag to REPCA

### PR DESCRIPTION
**Pull request by rodrigomha**
To do:
- [x] Test
- [ ] Docs of available flags

### CODE REVIEW

**Comment by jd-lara** [comment]
Is the change in the flags affecting the execution time?

File: src/initialization/inverter_components/init_outer.jl [hunk 1]
```diff
@@ -308,6 +308,30 @@ function initialize_outer!(
...
+        K_c = PSY.get_K_c(reactive_power_control)
+        R_c = PSY.get_R_c(reactive_power_control)
+        X_c = PSY.get_R_c(reactive_power_control)
+        VC_Flag = PSY.get_VC_Flag(reactive_power_control)
+        V_reg = sqrt(V_R^2 + V_I^2)
+        #Compute input to the compensated voltage filter
+        if VC_Flag == 0
+            V_flt_input = V_reg + K_c * q_elec_out
+        else
+            #V_flt_input = | V_reg - (R_c + jX_c)(I_r + jI_i) |
```
Comment (jd-lara) [inline src/initialization/inverter_components/init_outer.jl:323]: Why is this commented out?

File: src/initialization/inverter_components/init_outer.jl [hunk 2]
```diff
@@ -308,6 +308,30 @@ function initialize_outer!(
...
+            V_flt_input = sqrt(
```
Comment (jd-lara) [inline src/initialization/inverter_components/init_outer.jl:324]: Looks like this line isn't being hit.

File: src/models/inverter_models/outer_control_models.jl [hunk 3]
```diff
@@ -298,6 +299,110 @@ function _mdl_ode_RE_reactive_controller_AB!(
     inner_vars[Iq_oc_var] = Q_ext / max(Vt_filt, VOLTAGE_DIVISION_LOWER_BOUND)
 end
 
+# VC_Flag == 0 or 1 && Ref_Flag == 1 && PF_Flag == 0 && V_Flag == 1
```
Comment (jd-lara) [inline src/models/inverter_models/outer_control_models.jl:302]: We should add comments about the valid flags in the docs.

File: src/models/inverter_models/outer_control_models.jl [hunk 4]
```diff
@@ -298,6 +299,110 @@ function _mdl_ode_RE_reactive_controller_AB!(
...
+    V_cflt = reactive_controller_states[1]
+    ξq_oc = reactive_controller_states[2]
+    q_LL = reactive_controller_states[3]
+    ξ_Q = reactive_controller_states[4]
+
+    #Compute input to the compensated voltage filter
+    if VC_Flag == 0
+        V_flt_input = V_reg + K_c * q_elec_out
+    else
+        #V_flt_input = | V_reg - (R_c + jX_c)(I_r + jI_i) |
```
Comment (jd-lara) [inline src/models/inverter_models/outer_control_models.jl:375]: same here, why is it commented out?

File: src/initialization/inverter_components/init_outer.jl (hunk 2 above)
Comment (rodrigomha) [reply src/initialization/inverter_components/init_outer.jl:324]: We don't have a case for that case using VC_Flag, so we need to set-up a case with that. In any case, VC_Flag, is used for compensating when you are doing voltage regulation outside your local bus, and we currently do not support that.

File: src/initialization/inverter_components/init_outer.jl (hunk 1 above)
Comment (rodrigomha) [reply src/initialization/inverter_components/init_outer.jl:323]: I'll modify this to be a comment of the next line.

That was before when I was using the absolute value of complex numbers, the line that is after that, is the absolute value without using it.

File: src/models/inverter_models/outer_control_models.jl (hunk 3 above)
Comment (rodrigomha) [reply src/models/inverter_models/outer_control_models.jl:302]: Yes, good idea. Probably a paragraph in the Industrial Models docs

**Pull request merged**